examples/web-wagmi/src/publications/UsePublications.tsx

```diff
@@ -24,6 +40,28 @@ export function UsePublications() {
         <code>usePublications</code>
       </h1>
       <div>
+        <fieldset>
+          <legend>Publication types</legend>
+          {allPublicationTypes.map((value) => (
+            <label key={value}>
+              <input
+                type="checkbox"
+                checked={publicationTypes.includes(value)}
+                name="publicationTypes"
+                value={value}
+                onChange={(e) => {
+                  if (e.target.checked) {
+                    setPublicationTypes([...publicationTypes, value]);
```

Comment:
nit: could be a good use case for js `Set` given we can't have duplicates in the array (or at least we don't want :D)

Reply:
do you mean sth like below? I don't think it's a case here as the value can only be set from a select box, and it can't be checked twice

```ts
setPublicationTypes(Array.from(new Set([...publicationTypes, value])));
```